drivers/timer/nrf_rtc_timer.c

```diff
@@ -211,6 +211,24 @@ void _timer_idle_enter(s32_t sys_ticks)
 
 
 #ifdef CONFIG_TICKLESS_KERNEL
+/*
+ * Set RTC Counter Compare (CC) register to max value.
+ */
+static inline void program_max_cycles(void)
```

Comment:
General question: Why we are programming CC to specific value when no specific time-out is requested?  Coulnd't we just disable CC event and just rely on OVERFLOW to update tick count?

Reply:
In both cases, interrupt will be triggered. In case of CC value setting also,  we are actually checking for the ticks remaining to hit overflow. I am trying to make minimal changes to the existing driver which replies on CC events as the primary interrupt mechanism.